### PR TITLE
Simplify the update_ami code

### PR DESCRIPTION
**Pull request by bburtin**
Simplify the update_ami code and remove a bunch of (hopefully)
unnecessary logic.

Hardcode the Metavisor and guest root device names, instead of reading
them from the block device mappings.  We don't expect these to change
any time soon.

Consolidate the hacks that preserve volume settings, such as volume type
and IOPS, so that they don't obfuscate the rest of the code.

### CODE REVIEW

**Comment by bburtin** [comment]
I'm testing with Ubuntu and RHEL.  Let me know if there are any other AMIs that I should test.

File: brkt_cli/aws/update_ami.py [hunk 1]
```diff
@@ -146,23 +142,6 @@ def update_ami(aws_svc, encrypted_ami, updater_ami, encrypted_ami_name,
             else:
                 os.environ['NO_PROXY'] = updater.private_ip_address
 
-        enc_svc = enc_svc_class(host_ips, port=status_port)
-        log.info('Waiting for updater service on %s (port %s on %s)',
-                 updater.id, enc_svc.port, ', '.join(host_ips))
-        try:
-            wait_for_encryptor_up(enc_svc, Deadline(600))
```
Comment (apoorva-jindal) [inline brkt_cli/aws/update_ami.py]: Are you sure this always works? Metavisor in updater mode writes information to the disk like any new JWT provided etc, and when everything is done, it declares encryption complete. If you stop the instance immediately after, not sure how you will ensure that all processing MV needs to do in updater mode is complete?

Comment (bburtin) [inline brkt_cli/aws/update_ami.py]: Good call.  I'm in the process of putting this code back.  It worked ok when I tested on Ubuntu, but I hit a race condition with RHEL.

**Comment by bburtin** [comment]
After reinstating the code that waited for the encryption service, I was able to successfully encrypt and update Ubuntu, RHEL, CentOS, and Amazon Linux images.  Can someone approve if the code looks reasonable?

**Pull request merged**